### ISSUE TITLE
Project Lookup Session(s)-1

### ISSUE THREAD

**Issue by MuhammadAffanUsmani** [Muhammad-Hassan-Farooq/SE-Project-repo#33]
Aim: to improve the process. There will be brainstorming and project development sessions held as per-needs

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1200179730969548/1200230772756203) by [Unito](https://www.unito.io/learn-more)
┆Attachments: <a href="https://app.asana.com/app/asana/-/get_asset?asset_id=1200230722596971">Screenshot 2021-04-22 at 5.23.33 AM.png</a>

**Comment by Muhammad-Hassan-Farooq** [comment]
Meeting in session 


<img width="1433" alt="Screenshot 2021-04-22 at 5 23 33 AM" src="https://user-images.githubusercontent.com/81969481/115638520-fee0d200-a32b-11eb-9475-89faabbd8977.png">

**Comment by Muhammad-Hassan-Farooq** [comment]
We also noted the contributions and the file authors in this lookup-session and also prepared all the necessary documentation material for the files present at the directory. We also noted our backlog details and also looked up all the present streams to re-check if there was any merge or pull not created or successful.

<img width="1203" alt="Screenshot 2021-04-22 at 5 42 57 AM" src="https://user-images.githubusercontent.com/81969481/115639306-b2969180-a32d-11eb-8bf3-5fa65e375852.png">

**Comment by Muhammad-Hassan-Farooq** [comment]
At-last this Project lookup-session was a success from these perspectives - 

- Back-end-development | Database-coding | Partial Front-end-development
- Also we have checked our Asana Project board all the tasks were successfully synchronised and also maintained in an ethical decorum by their assignees and agile coach @Muhammad-Hassan-Farooq 
- All of the discussion today was consisting the Project team-Leader @MuhammadAffanUsmani as well, important matters were discussed and improvement suggestions were passed from the agile coach to the project leader.
- Moving on to the next stage of our project as we have uploaded the main stream and structural-coding for our project we will be proceeding to the (version-s) analyses for our next phase of our GitHub management.
- New backlog will be decided today by the evening and the next phase will start when the analyses are finalised with all the other team-members.
- Notting this that no problem or issue was in-sight during this lookup-session, everything was smooth till now and the contribution status and update is (attached above) all the of the contributors were fine and were on-time with their tasks.
